wikimedia/wikipedia-ios
# [HOLD] Watchlist - Show empty Watchlist view when tapped in Account Settings

**Phabricator:** https://phabricator.wikimedia.org/T335606 (partially)

### Notes
Shows empty Watchlist view when tapped from Account Settings. The Components dependency here currently points to the latest in the Components PR waiting for review. Note that the theme isn't currently passed along to the Components framework yet, and thus the Watchlist view itself will not style/color appropriately. This will be addressed in future PRs.

This shouldn't be merged until 1. the paired Components PR is merged and 2. the Components dependency is updated to the new commit.

### Test Steps
1. Log in to an account
2. Tap Watchlist in Account Settings, confirm empty new view opens
3. Change your theme manually
4. Return to Watchlist, confirm the navigation bar is the appropriate color for your current theme
